packages/interactor/test/integrations/cypress/integration/interactor.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { Heading } from '../../../../src';
+
+describe('Cypress with Interactors', () => {
+  it('', () => {
+    cy.visit('/')
+    cy.do(Heading('todos').exists())
```

Comment:
Technically here we want to check a "step" in a `do` and an assertion in an `expect` and make sure they work when used correctly, but that they fail if you use a step outside of a `do`. 

In this example, I'd use `.expect()`

Reply:
Oh so the test looks like that right now because setting `bigtestGlobals.document = "step"` for `cy.do()` does not return an error when you pass in an assertion so I'm trying to figure out why.

Reply:
It's ok to pass an assertion to `do`, what's not ok is to pass a step to `expect`